### PR TITLE
sync Notification API

### PR DESCRIPTION
**Pull request by skyclouds2001**

### CODE REVIEW

File: files/zh-cn/web/api/notifications_api/index.md [hunk 1]
```diff
@@ -3,43 +3,50 @@ title: Notifications API
...
-从这里，用户可以选择允许来自此来源的通知，阻止来自此来源的通知，或不选择此点。一旦做出选择，该设置通常将持续用于当前会话。
+这将产生一个请求对话框，内容如下：
 
-> **备注：** 从 Firefox 44 开始，通知（Notifications）和推送（[Push](/zh-CN/docs/Web/API/Push_API)）的权限已合并。如果为通知授予权限，推送也将启用。
+![一个对话框，要求用户允许来自该源的通知。有一些选项可以禁止或允许通知](screen_shot_2019-12-11_at_9.59.14_am.png)
 
-接下来，使用 {{domxref("Notification.Notification","Notification()")}} 构造函数创建一个新通知。这个方法可以传入两个参数。这必须传递一个标题参数，并可以选择性地传递一个选项对象来指定选项，如文本方向，正文，显示图标，通知声音播放，等等。
+从这里，用户可以选择允许来自此来源的通知或阻止来自此来源的通知。一旦做出选择，该设置通常将持续用于当前会话。
+
+> **备注：** 从 Firefox 44 开始，通知（Notifications）和推送（[Push](/zh-CN/docs/Web/API/Push_API)）的权限已合并。如果为通知授予权限，推送也将启用。
```
Comment (yin1999) [inline files/zh-cn/web/api/notifications_api/index.md:29]: It should be ok to remove the `s`

```suggestion
> **备注：** 从 Firefox 44 开始，通知（Notification）和推送（[Push](/zh-CN/docs/Web/API/Push_API)）的权限已合并。如果为通知授予权限，推送也将启用。
```

File: files/zh-cn/web/api/notifications_api/index.md [hunk 2]
```diff
@@ -3,43 +3,50 @@ title: Notifications API
...
 
-{{AvailableInWorkers}}
+接下来，使用 {{domxref("Notification.Notification","Notification()")}} 构造函数创建一个新通知。这个函数必须传递一个标题参数，并且可以选择传递一个选项对象来指定选项，例如文本方向、正文、要显示的图标、要播放的通知声音等等。
 
-此外， **Notifications API** 规范对 [ServiceWorker API](/zh-CN/docs/Web/API/ServiceWorker_API)指定了多个添加，以允许 service workers 发送通知。
+此外， **Notifications API** 规范对 [ServiceWorker API](/zh-CN/docs/Web/API/ServiceWorker_API) 指定了多个添加，以允许 Service Worker 发送通知。
```
Comment (yin1999) [inline files/zh-cn/web/api/notifications_api/index.md:33]:
```suggestion
此外， Notifications API 规范对 [ServiceWorker API](/zh-CN/docs/Web/API/ServiceWorker_API) 指定了多个添加，以允许 Service Worker 发送通知。
```

File: files/zh-cn/web/api/notifications_api/index.md [hunk 3]
```diff
@@ -3,43 +3,50 @@ title: Notifications API
...
 
-> **备注：** 想了解怎么在你的应用里使用通知接口，请阅读 [Using the Notifications API](/zh-CN/docs/Web/API/Notifications_API/Using_the_Notifications_API)。
+> **备注：** 想了解怎么在你的应用里使用通知接口，请阅读 [使用 Notifications API](/zh-CN/docs/Web/API/Notifications_API/Using_the_Notifications_API)。
```
Comment (yin1999) [inline files/zh-cn/web/api/notifications_api/index.md:35]:
```suggestion
> **备注：** 想了解怎么在你的应用里使用通知接口，请阅读[使用 Notifications API](/zh-CN/docs/Web/API/Notifications_API/Using_the_Notifications_API)。
```

File: files/zh-cn/web/api/notifications_api/index.md [hunk 4]
```diff
@@ -3,43 +3,50 @@ title: Notifications API
...
 
-## 接口（Notifications interfaces）
+## 接口
```
Comment (yin1999) [inline files/zh-cn/web/api/notifications_api/index.md:37]:
```suggestion
## Notifications 接口
```

File: files/zh-cn/web/api/notifications_api/index.md [hunk 5]
```diff
@@ -3,43 +3,50 @@ title: Notifications API
...
 
 - {{domxref("Notification")}}
-  - : 定义的通知对象
+  - : 定义的通知对象。
 
-### 附加参数（Service worker additions）
+### 附加参数
```
Comment (yin1999) [inline files/zh-cn/web/api/notifications_api/index.md:42]:
```suggestion
### Service worker 附加内容
```

File: files/zh-cn/web/api/notifications_api/screen_shot_2019-12-11_at_9.59.14_am.png
[the remark this answers is not in the document]
Comment (skyclouds2001) [reply files/zh-cn/web/api/notifications_api/screen_shot_2019-12-11_at_9.59.14_am.png:1]: Got it!

File: files/zh-cn/web/api/notifications_api/index.md [hunk 6]
```diff
@@ -3,43 +3,50 @@ title: Notifications API
...
-  - : 定义的通知对象
+  - : 定义的通知对象。
 
-### 附加参数（Service worker additions）
+### Service worker 附加内容
 
 - {{domxref("ServiceWorkerRegistration")}}
   - : 该对象包含 {{domxref("ServiceWorkerRegistration.showNotification()")}} 和 {{domxref("ServiceWorkerRegistration.getNotifications()")}} 方法，用于控制通知的显示。
 - {{domxref("ServiceWorkerGlobalScope")}}
   - : 该对象包含 {{domxref("ServiceWorkerGlobalScope.onnotificationclick")}} 事件处理函数，用于通知被点击的时候触发绑定的事件处理函数。
```
Comment (yin1999) [inline files/zh-cn/web/api/notifications_api/index.md:47]:
```suggestion
  - : 该对象包含 {{domxref("ServiceWorkerGlobalScope.notificationclick_event", "onnotificationclick")}} 事件处理器，用于通知被点击的时候触发自定义函数。
```

**Review by yin1999 — APPROVED** [review]

**Comment by yin1999** [comment]
Thanks @skyclouds2001 :+1:

**Pull request merged**